core/ui/EditTemplate/fields.tid

```diff
@@ -10,14 +10,20 @@ $:/config/EditTemplateFields/Visibility/$(currentField)$
 [[hide]] -[title{$(config-title)$}]
 \end
 
+<!-- Beware this is duplicated from EditTemplate.tid. For details see bug #7054 -->
```

Comment:
TW allows you to use the `\import` pragma. Would this be an option to avoid code duplication? https://tiddlywiki.com/prerelease/#Pragma

Reply:
Hello @pmario,

the problem would be that if a plugin or user has overwritten the source tiddler (in this case `FieldEditor.tid`) the `\import` paragma would not find the imported macro, and we would be back to square one.

This would only work if I create a completely new tiddler (that is extremely unlikely to be already overwritten) and import it into both of the destinations. But I think this is too much of a change for a bugfix.

Maybe it would be a good idea to define a path in TiddlyWiki that must not be overwritten by the user or plugins, where we can put utility macros to import without the risk that they vanish.

Reply:
>the problem would be that if a plugin or user has overwritten the source tiddler (in this case FieldEditor.tid) the \import paragma would not find the imported macro, and we would be back to square one.

Ahhh ... OK